src/com/opc_ua/opcua_action_info.h

```diff
@@ -46,11 +46,26 @@ class CActionInfo {
     /**
      * 2-Tuple to reference a specific node in an OPC UA server composed of the node ID and browsepath of the node
      */
-    struct CNodePairInfo {
+    class CNodePairInfo {
+      public:
         CNodePairInfo(UA_NodeId *paNodeId, const std::string &paBrowsePath) :
-            mNodeId(paNodeId), mBrowsePath(paBrowsePath) {
+            mNodeId(paNodeId, UA_NodeId_delete), mBrowsePath(paBrowsePath) {
+        }
+
+        UA_NodeId *getNodeId() const {
+          return mNodeId.get();
+        }
+
+        void setNodeId(UA_NodeId * paNodeId){
```

Comment:
There is a blank space between the asterisk and the parameter name.
```suggestion
        void setNodeId(UA_NodeId *paNodeId){
```

Reply:
changed